lib/public/components/common/form/inputs/DateTimeInputComponent.js

```diff
@@ -172,17 +174,18 @@ export class DateTimeInputComponent extends StatefulComponent {
             return null;
         }
 
-        const minDateDayAfter = new Date(min + MILLISECONDS_IN_ONE_DAY);
+        const rawDate = raw.date || null;
+        const rawTime = raw.time || null;
 
         const minDateAndTime = formatTimestampForDateTimeInput(min, this._seconds);
         const ret = {};
 
-        if (raw.date !== null && raw.date === minDateAndTime.date) {
+        if (rawDate !== null && rawDate === minDateAndTime.date) {
             ret.time = minDateAndTime.time;
         }
 
-        if (raw.time !== null && raw.time < minDateAndTime.time) {
-            ret.date = formatTimestampForDateTimeInput(minDateDayAfter.getTime(), this._seconds).date;
+        if (rawTime !== null && rawTime < minDateAndTime.time) {
+            ret.date = formatTimestampForDateTimeInput(min + MILLISECONDS_IN_ONE_DAY, this._seconds).date;
         } else {
```

Comment:
maybe `minTimestamps` or something instead of `min`, the same with `max`

Reply:
Right, done